docs/sources/next/using-k6/modules.md

```diff
@@ -65,282 +67,101 @@ export default function () {
+### Extension modules
 
-#### Configuring Webpack
+Like the [k6 APIs](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api), you can build custom modules in Go code and expose them as JavaScript modules. These custom Go-to-JS modules are known as [k6 extensions](https://grafana.com/docs/k6/<K6_VERSION>/extensions).
 
-Once these packages have been added, the next step will be to set up a `webpack.config.js` file:
-
-{{< code >}}
+Below is an example that imports the `k6/x/kubernetes` module from the [xk6-kubernetes](https://github.com/grafana/xk6-kubernetes) extension.
 
 ```javascript
-const path = require('path');
-
-module.exports = {
-  mode: 'production',
-  entry: {
-    login: './src/login.test.js',
-    signup: './src/signup.test.js',
+import { Kubernetes } from 'k6/x/kubernetes';
+
+const podSpec = {
+  apiVersion: 'v1',
+  kind: 'Pod',
+  metadata: { name: 'busybox', namespace: 'testns' },
+  spec: {
+    containers: [
+      {
+        name: 'busybox',
+        image: 'busybox',
+        command: ['sh', '-c', 'sleep 30'],
+      },
+    ],
   },
-  output: {
-    path: path.resolve(__dirname, 'dist'), // eslint-disable-line
-    libraryTarget: 'commonjs',
-    filename: '[name].bundle.js',
-  },
-  module: {
-    rules: [{ test: /\.js$/, use: 'babel-loader' }],
-  },
-  target: 'web',
-  externals: /k6(\/.*)?/,
 };
+export default function () {
+  const kubernetes = new Kubernetes();
+  kubernetes.create(podSpec);
+  const pods = kubernetes.list('Pod', 'testns');
+  pods.map(function (pod) {
+    console.log(pod.metadata.name);
+  });
+}
 ```
 
-{{< /code >}}
-
-`Mode`
-
-Tells Webpack to automatically use the optimizations associated with the `mode`.
-Additional details available in [the webpack docs](https://webpack.js.org/configuration/mode/).
+How do k6 extensions (Go-to-JS modules) work? For enhanced performance, the k6 engine is written Go and embeds a JavaScript VM ([goja](https://github.com/dop251/goja)) to execute JavaScript test code. Thus, you can build your modules in Go code and import them as JavaScript as usual.
 
-`Entry`
+To learn more about using or creating k6 extensions, refer to the [Extension documentation](https://grafana.com/docs/k6/<K6_VERSION>/extensions).
 
-The files Webpack will use as its entry points while performing the bundling. From these points,
-Webpack will automatically traverse all imports recursively until every possible dependency path has
-been exhausted. For instance:
+## Sharing JavaScript modules
 
-{{< code >}}
-
-```javascript
-// login.test.js
-
-import { SomeService } from './some.service.js';
+As mentioned previously, users can import custom Javascript libraries by loading either local or remote modules. Thus, we have two options to import JavaScript modules, along with various methods to distributed them.
```

Comment:
```suggestion
As mentioned previously, users can import custom JavaScript libraries by loading either local or remote modules. Because of that, we have two options to import JavaScript modules, along with various methods to distribute them.
```